src/lib/progress.c

```diff
@@ -169,6 +169,13 @@ void progress_set_spinner(bool status_flag)
 	}
 }
 
+void progress_report_interdeterminate()
```

Comment:
"interdeterminate"? is that intended? or did you mean indeterminate?

Reply:
Ohh Thats a typo...indeterminate :P